napari/layers/tracks/tracks.py

```diff
@@ -194,11 +209,9 @@ def _get_state(self):
                 'data': self.data,
                 'properties': self.properties,
                 'graph': self.graph,
-                'display_id': self.display_id,
-                'display_tail': self.display_tail,
-                'display_graph': self.display_graph,
```

Comment:
@quantumjot For better or for worse the keys of the `_get_state` dictionary are checked against input parameters to the function. These there are missing, as they are the display ones - is it ok if we drop them from this function? We want to improve the `_get_state` method soon, see discussion in #1636

Reply:
I don't see why removing them would be a problem.